slip-0044.md

```diff
@@ -1352,6 +1352,8 @@ All these constants are used as hardened derivation.
 | 608589380  | 0xa4465644                    | FVDC    | ForumCoin                         |
 | 1179993420 | 0xc655454c                    |         | Fuel                              |
 | 1179993421 | 0x80233434                    | TTNC    | TakeTitan                         |
+| 1179993441 | 0x80233444                    | QFS     | Qfs                               |
```

Comment:
1. keep the list sorted
2. make sure the dec values match the hex values
3. please fix also TTNC value which is wrong